packages/server/lib/helpers/email.ts

```diff
@@ -0,0 +1,22 @@
+import { basePublicUrl } from '@nangohq/utils';
+import EmailClient from '../clients/email.client.js';
+
+export function sendVerificationEmail(email: string, name: string, token: string) {
+    const emailClient = EmailClient.getInstance();
+    emailClient.send(
+        email,
+        `Verify your email address`,
+        `
+<p>Hi ${name},</p>
+
+<p>We need to verify your email address ${email}</p>
+
+<p>Please verify this account by clicking <a href="${basePublicUrl}/signup/verification/${token}">here</a></p>
```

Comment:
nitpick: I would show the full url instead of `here` just to make it clearer it is going to our domain and it is not some sort of phishing attempt